alloy_python/uapi/metadata.py

```diff
@@ -0,0 +1,37 @@
+# uapi/metadata.py
+import requests
+from ..constants import BASE_URL
+
+class Metadata:
+    def __init__(self, api_key):
+        if not api_key:
+            raise ValueError("API key must be provided")
+
+        self.api_key = api_key
+        self.headers = {
+            'Authorization': f'Bearer {api_key}',
+            'accept': 'application/json'
+        }
+        self.base_url = BASE_URL
+
+    def list_operations(self, app):
+        url = f'{self.base_url}/one/metadata/operations/?app={app}'
+        return self._api_request('GET', url)
+
+    def list_fields(self, app, operation):
+        url = f'{self.base_url}/one/metadata/fields?app={app}&operation={operation}'
+        return self._api_request('GET', url)
+
+    def _api_request(self, method, url, params=None):
+        try:
+            response = requests.request(
+                method,
+                url,
+                headers=self.headers,
+                params=params
+            )
+            response.raise_for_status()
```

Comment:
Same here

Reply:
Updated classes to align with existing error handling for CRM, Accounting, Commerce. 

The raise_for_status() is a trigger for the except blocks, which have the acutual error handling logic. 

Currently, these return the response body JSON as received from the API (error message) if there's any error
![image](https://github.com/alloy-automation/alloy-python/assets/28990947/f247e768-a95b-4958-980a-5f3bb25af715)